internal_test_data/network_config_interface_not_defined.yaml

```diff
@@ -0,0 +1,24 @@
+test_duration: 5
```

Comment:
Really 5?

Reply:
those are config files used for unittests.... it's not really gonna be used in an actual longevity